spinn_front_end_common/utilities/globals_variables.py

```diff
@@ -205,15 +203,17 @@ def run_report_directory():
     This will be the path used by the last run call or to be used by
     the next run if it has not yet been called.
 
+    .. note:
+        The behaviour when called before setup is subject to change
+
     :rtpye: str
-    :raises ValueError:
-        if the system is in a state where path can't be retrieved
+    :raises SimmulatorNotSetupException:
+        if the system has a status where path can't be retrieved
```

Comment:
It still raises ValueError (also Simmulator --> Simulator)

Reply:
Oops fixed